plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/InvisibleCommand.java

```diff
@@ -1,60 +1,79 @@
 package com.denizenscript.denizen.scripts.commands.entity;
 
 import com.denizenscript.denizen.nms.NMSHandler;
-import com.denizenscript.denizen.utilities.Utilities;
-import com.denizenscript.denizen.utilities.debugging.Debug;
 import com.denizenscript.denizen.npc.traits.InvisibleTrait;
 import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.NPCTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizen.utilities.packets.NetworkInterceptHelper;
 import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
 import com.denizenscript.denizencore.objects.Argument;
 import com.denizenscript.denizencore.objects.core.ElementTag;
 import com.denizenscript.denizencore.scripts.ScriptEntry;
 import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
-import net.citizensnpcs.api.npc.NPC;
-import org.bukkit.entity.ArmorStand;
-import org.bukkit.entity.ItemFrame;
-import org.bukkit.potion.PotionEffect;
+import org.bukkit.entity.*;
 import org.bukkit.potion.PotionEffectType;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+
 public class InvisibleCommand extends AbstractCommand {
 
     public InvisibleCommand() {
         setName("invisible");
-        setSyntax("invisible [<entity>] (state:true/false/toggle)");
-        setRequiredArguments(1, 2);
+        setSyntax("invisible [<entity>] (state:true/false/toggle/reset) (for:<player>|...)");
+        setPrefixesHandled("for");
+        setRequiredArguments(0, 3);
         isProcedural = false;
     }
 
     // <--[command]
     // @Name Invisible
-    // @Syntax invisible [<entity>] (state:true/false/toggle)
-    // @Required 1
-    // @Maximum 2
-    // @Short Makes an NPC or entity go invisible.
+    // @Syntax invisible [<entity>] (state:true/false/toggle/reset) (for:<player>|...)

```

Comment:
Syntax meta needs to be updated if entity is now optional

Reply:
Changed to `(<entity>)`